tools/cachegrand-benchmark-generator/src/output.c

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (C) 2018-2022 Vito Castellano
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the BSD license.  See the LICENSE file for details.
+ **/
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+#include <json.h>
+
+#include "matcher.h"
+#include "builder.h"
+#include "analyzer.h"
+#include "support.h"
+
+#include "output.h"
+
+void output_stdout_print(
```

Comment:
Wrap debug code in a #if DEBUG == 1 .. #endif so will be compiled only when in debug mode